r2dbc-examples/src/test/java/com/nebhale/r2dbc/examples/PostgresqlServerResource.java

```diff
@@ -110,7 +110,7 @@ String getUsername() {
     }
 
     private static Path getCachePath() {
-        return Paths.get("target/postgresql");
```

Comment:
What exactly is this change achieving? Changing the directory from a place where all builds are already putting files to a temporary directory doesn’t seem like a Linux-specific change.

Reply:
This change is not Linux specific, but makes tests working on Linux (otherwise they fail with a `error while loading shared libraries: libicuuc.so.53: cannot open shared object file: No such file or directory` message) as described in this comment https://github.com/yandex-qatools/postgresql-embedded/issues/110#issuecomment-346510808. I admit I have not spent time to analyze why.